project/app/build.gradle

```diff
@@ -56,7 +56,7 @@ repositories {
 }
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion 25
```

Comment:
Why is it required?

Reply:
For now, because of the build tools version I am using currently, (v25) so they match the travis.yml file, so I had to change temporarily the compileSdk to 25, changing this to 26 causes the CI build to fail because mismatch between dependencies stablished in the build.gradle file of 'app' versus the travis.yml file.

Reply:
Ok, considering that we upgraded it from 23, "dowgrading" to 25 is ok.